integration_test/lib/vnspec/spec/simple_seg_spec.rb

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+require_relative "spec_helper"
+
+describe "simple_seg" do
```

Comment:
I think this abbreviation is quite obvious enough. When other developers look at the integration test, they won't immediately know that seg means segment. I think it's better to write "simple_segment" without abbreviating.